lib/customer-deposit-wallet/src/Cardano/Wallet/Deposit/Read.hs

```diff
@@ -59,6 +60,7 @@ import qualified Cardano.Wallet.Primitive.Types.Tx.TxIn as W
 import qualified Cardano.Wallet.Primitive.Types.Tx.TxOut as W
 import qualified Cardano.Wallet.Primitive.Types.UTxO as W
 import qualified Data.ByteString as BS
+import qualified Data.ByteString.Char8 as B8
 -- import qualified Ouroboros.Consensus.Cardano.Block as O
```

Comment:
I hope we will clean those things later... Now, not a good moment?

Reply:
Yes. `Cardano.Wallet.Deposit.Read` is essentially a messy lumber room at the moment. 😅